src/Tizen.Multimedia.Remoting/WebRTC/MediaSource.cs

```diff
@@ -213,6 +213,70 @@ public Size VideoResolution
             }
         }
 
+        /// <summary>
+        /// Gets or sets the video frame rate of the current media source.
+        /// </summary>
+        /// <remarks>
+        /// This API is only supported in video media source, especially <a cref="MediaCameraSource"/>,
+        /// <a cref="MediaScreenSource"/>, <a cref="MediaTestSource(MediaType)"/>.<br/>
+        /// </remarks>
+        /// <value>A value that specifies the video frame rate.</value>
+        /// <exception cref="ArgumentException">VideoFrameRate is less than zero.</exception>
```

Comment:
suggestion: I think that 'less than or equal to zero' is right.

Reply:
Done